Makefile

```diff
@@ -148,7 +154,9 @@ superuser: ## create a DjangoCMS superuser
 .PHONY: superuser
 
 test-back: ## run back-end tests
-	@$(COMPOSE_TEST_RUN_APP) pytest
+	@$(COMPOSE_TEST) kill db
+	@$(COMPOSE_TEST) rm -f db
```

Comment:
actually maybe I should call bin/pytest in this make command because the switch is made more elegantly. 
Or should we remove this make command since it is not optimal and forces to recreate the db each time? Let's push the bin scripts to the light WDYT?